core/domain/rights_manager.py

```diff
@@ -644,6 +644,21 @@ def _publish_activity(committer_id, activity_id, activity_type):
         committer_id, activity_id, activity_type, ACTIVITY_STATUS_PUBLIC,
         '%s published.' % activity_type)
 
+    # Add contribution dates to the owners and editors
```

Comment:
... and editors. This assumes that the only people who could possibly make commits to the exploration are those listed explicitly as owners/editors of the exploration.

(This may change down the road, so it seems better to be clear here.)